source/agora/consensus/data/Block.d

```diff
@@ -439,7 +439,7 @@ unittest
 *******************************************************************************/
 
 public Block makeNewBlock (Transactions)(const ref Block prev_block,
-    Transactions txs, Enrollment[] enrollments) @safe nothrow
+    Transactions txs, Enrollment[] enrollments) @safe
```

Comment:
Ditto comment about https://github.com/bpfkorea/agora/pull/1125 being merged, so I think this should remain being `nothrow`.